server/cluster/cluster.go

```diff
@@ -56,6 +57,12 @@ var backgroundJobInterval = 10 * time.Second
 const (
 	clientTimeout              = 3 * time.Second
 	defaultChangedRegionsLimit = 10000
+	progressMetricsTimer       = 5  // progress run interval second
+	progressMaxRetryLimit      = 20 // max check times
+
+	progressFinish           = 1 // progress progress finish
+	progressStoreOffLine     = 2 // store offline
```

Comment:
What does `StoreOffline` mean?

Reply:
StoreOffline: the store may not in raft cluster,some condition is:  
up store may not be  in raft cluster ,ig:
1. pick storeA
2. storeA is deleted by someone 
down: 
1. the offline store is not exist